website/developer-docs/api/flow-executor.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Flow executor
+title: Flow executor (backend)
 ---
 
-A big focus of authentik is the flows system, which allows you to combine and build complex conditional processes using stages and policies. Normally, these flows are executed in the browser using the authentik inbuilt flow executor (/if/flows).
```

Comment:
You know, I don't think we need to use "inbuilt" or "default."  Something that suggests what it's _for_ rather than its position in a hierarchy of executors.  "...using authentik's standard browser-based flow executor..." maybe?  That way the rest of the conversation can be about _non-standard_ (SFE) or non-browser (headless) executors.